src/pages/api/public/metrics/daily.ts

```diff
@@ -149,6 +150,12 @@ export default async function handler(
     }
   } catch (error: unknown) {
     console.error(error);
+    if (isPrismaException(error)) {
+      return res.status(500).json({
+        message: "Error processing events",
```

Comment:
the message does not make sense in this case. I'd just suggest to remove it alltogether in all routes as it does not add much detail/context